7DOF free index selection (#274)
ros-planning/moveit_tutorials: doc/ikfast/ikfast_tutorial.rst

 export BASE_LINK="0"
 export EEF_LINK="8"

> If you have a 7 DOF arm you will need to specify a free joint. Selecting the correct free joint for a 7 DOF robot can be more of an art than a science and have significant impact on peformance of your kinematics plugin. We suggest experimenting with different choices ::

Author: ```suggestion
If you have a 7 DOF arm you will need to specify a free joint. Selecting the correct free joint for a 7 DOF robot can have significant impact on performance of your kinematics plugin. We suggest experimenting with different choices for the ``FREE_INDEX`` value ::
```
